generalTable.Rmd

```diff
@@ -5,7 +5,7 @@ output: html_document
 
 ### **About**
 
-This website contains a list of general informatics platforms funded by ITCR grants. General platforms are those which work with a wide variety of data types, from imaging data to "omics" data. Each resource will have up to three links that navigate towards their main site, their documentation, and a list of their publications.
+This website contains a list of general informatics platforms funded by ITCR grants. General platforms are those which work with a wide variety of data types, from imaging data to “omics” data. Each resource will have up to three links that navigate towards their main site, their documentation, and a list of their publications.
```

Comment:
I think we need to define what we mean by platform. This is a tricky thing to describe! 

Web-based computing platform might be the right term - I like that we are going to call the tab "Computing resources", as this will go well with "Data Resources". We will also in the future want a tab for tables related to more individual analysis tools, maybe we will call that "Software Resources" but idk as these computing resources are also software... but they are also more than that - so maybe this makes sense?

Reply:
I agree. But we can definitely discuss more about the labels and descriptions once we find a good website structure that we are happy with. This is something I just put in as a placeholder.

Reply:
Oh I understand that you just put that in as a placeholder. I was trying to give you more information about what we can say in terms of describing what the tables actually show.

Reply:
Yes, we can wait - but I wanted to give you that information in case you need it with your next pull request after you rearrange

Reply:
I see that you already did some rearrangements which look quite nice

Reply:
Yes, understood. These are really great feedback! Thank you! 
 I just did not want you to think this was my best effort, haha.